src/components/ui/PressCardList/Item/PressCardItem.module.css

```
@@ -0,0 +1,75 @@
+.root {
+  margin-bottom: 50px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 24px;
+  position: relative;
+}
+
+.header,
+.body {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.body {
+  gap: 30px;
+}
+
+.date {
+  font-weight: 500;
+  font-size: 12px;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  color: #aaa078;
+}
+
+.title {
+  font-weight: 600;
+  font-size: 20px;
+  letter-spacing: -0.01em;
+  text-transform: uppercase;
+  color: #f8f7f0;

```

@Yura33-dev let's try to use our defined color variables wherever possible please. Those are stored at `src/app/globals.css` 😉
```suggestion
  color: rgb(var(--rgb-foreground));
```